### PR TITLE
Update communication section

### PR DESCRIPTION
**Pull request by durrantm**

### CODE REVIEW

**Comment by mikeraimondi** [comment]
:+1: LGTM

**Comment by bensaufley** [comment]
Can probably squash the commits?

File: protocol/communication/README.md [hunk 1]
```diff
@@ -1,9 +1,39 @@
...
+# Communication
 
 A guide for communicating within EverQuote.
 
-JIRA Tickets
--------
+### Feature / Bug tracking
 
-* People assign themselves to tickets. When in doubt, do not assign a ticket to someone.
+We use Jira at https://myquote.atlassian.net/
```
Comment (bensaufley) [inline protocol/communication/README.md]: Style nit on the style guides … how meta! The Guides tend to use this format for links in sentences:

```md
We use [JIRA]

(then in the foot:)

[JIRA]: https://myquote.atlassian.net
```

Same can be applied to Slack, Confluence… all three can be listed at the bottom of the MD doc

File: protocol/communication/README.md [hunk 2]
```diff
@@ -1,9 +1,39 @@
...
+
+### Daily Standup
+
+We meet daily in the morning to discuss status
+
+### Documentation
+
+We are starting to use Confluence at: https://myquote.atlassian.net/wiki/
+
+In the past we have used evernote and google docs
```
Comment (bensaufley) [inline protocol/communication/README.md]: I'm not sure we've quite phased out Evernote or Google Docs yet? Might be worth adding info about how to get set up with Evernote, and a link to drive.google.com?

Another small style nit: would like the Guides to recognize proper nouns as such, and use full sentences when reasonable.

File: protocol/communication/README.md [hunk 3]
```diff
@@ -1,9 +1,39 @@
...
+
+### Pairing
+
+Pairing takes place at developers desks as needed
+
+We use google hangouts for remote pairing
```
Comment (bensaufley) [inline protocol/communication/README.md]: Link to hangouts.google.com?

File: protocol/communication/README.md [hunk 4]
```diff
@@ -1,9 +1,39 @@
...
+
+### Code Reviews
+
+Code reviews are done through github pull requests
```
Comment (bensaufley) [inline protocol/communication/README.md]: Semantic note: I use "code review" as a singular. "Code review is done through Github pull requests." That's usually what I hear? I bet I'm the only one in the world who has an opinion on this? It's inconsequential, but the Guides are a place for discussion, right? Anyone else?

**Comment by bensaufley** [comment]
@durrantm Do you want to respond to/incorporate my comments? Would you prefer that I just tagged onto this PR?

**Comment by durrantm** [comment]
Completely forgot about this, sorry.  yeah that's fine.

**Comment by bensaufley** [comment]
@mikeraimondi @durrantm @kylepfeeley everyone OK with the changes I've made? Can I merge this? It's been open forever.

**Comment by mikeraimondi** [comment]
:+1:

**Pull request merged**